gitlabform/gitlabform/core.py

```diff
@@ -69,9 +74,26 @@ def method_wrapper(self, project_and_group, configuration):
 
 class GitLabFormCore(object):
 
-    def __init__(self):
-        self.args = self.parse_args()
-        self.set_log_level()
+    def __init__(self, project_or_group=None, config_string=None, debug=False):
+
+        if project_or_group and config_string:
+            self.project_or_group = project_or_group
+            self.config_string = config_string
+            self.verbose = False
+            if debug:
+                self.debug = True
+            else:
+                self.debug = False
```

Comment:
can it be just one line `self.debug = bool(debug)` ? `debug` is a default argument set to `False` anyway so this covers only case when something else than boolean is passed to the init

Reply:
Of course it can, thanks!